crates/router/src/compatibility/stripe/customers.rs

```diff
@@ -167,11 +167,11 @@ pub async fn customer_delete(
 pub async fn list_customer_payment_method_api(
     state: web::Data<routes::AppState>,
     req: HttpRequest,
-    path: web::Path<String>,
+    json_payload: web::Query<payment_methods::PaymentMethodListRequest>,
```

Comment:
Please don't change existing endpoint. It is meant to be stripe compatible https://stripe.com/docs/api/payment_methods/customer_list

Reply:
sure will change this